lib/ansible/plugins/lookup/sequence.py

```diff
@@ -33,6 +33,8 @@
         type: number
       format:
         description: return a string with the generated number formatted in
+    notes:
+      - This lookup is deprecated in favor of using "loop" and the "range" function
```

Comment:
where can I find info about the `range` function?  I don't see it on either of these pages:
https://docs.ansible.com/ansible/devel/plugins/lookup.html#plugin-list
https://docs.ansible.com/ansible/devel/user_guide/playbooks_filters.html

Reply:
http://jinja.pocoo.org/docs/dev/templates/#range

Reply:
great!  Can we say `jinja "range" function`?